assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -54,11 +55,18 @@ function useUploadVideoFrame({ updateMediaElement }) {
         featured_media: posterId,
         post: storyId,
       });
-      if (elementId) {
-        // TODO Update all elements that use video.
-        const newState = { resource: { ...resource, posterId, poster } };
-        setProperties(elementId, newState);
-      }
+      currentPage.elements.forEach((element) => {
```

Comment:
This should be part of the story reducer. It should also video all elements on all pages and update all of them.

Reply:
It makes sense for me. One thing here, we have updateElements method already but it supposed to update the elements for the currentPage. And also to use that methods, we need to extract all the elementIds from here to update all the elements. Instead I think it would be good to create new method. Just for now created `updateVideoElementsByVideoId`. Supposed to update all the elements through the pages with videoId